pages/common/git-difftool.md

```diff
@@ -0,0 +1,20 @@
+# git difftool
+
+> Show changes to files using external diff tools. Takes the same options and arguments as git diff.
```

Comment:
```suggestion
> Show file changes using external diff tools. Takes the same options and arguments as git diff.
```
What does the second sentence mean, though?

Reply:
It uses the same options and arguments as `git diff`..?

If `git diff --foo bar` is valid, then `git difftool --foo bar` is also valid.

Reply:
I think `Accepts` would be more clear than `takes` then. Just my take.